source/gui/installerGui.py

```diff
@@ -22,7 +23,18 @@
 import systemUtils
 import ui
 from NVDAState import WritePaths
-from .message import displayDialogAsModal
+from .message import DialogType, MessageDialog, ReturnCode, displayDialogAsModal
+
+IsUserAnAdmin = ctypes.windll.shell32.IsUserAnAdmin
```

Comment:
I think we should make this private, we may want to move this definition into somewhere more appropriate in future